codecov.yml

```diff
@@ -11,6 +11,10 @@ codecov:
       # Learn more at https://codecov.io/docs#yaml_default_commit_status
       project: true
 
+  status: 
+    changes: 
+      enabled: no
+
```

Comment:
There is already a `status` tag under `coverage` (Line 10), and `changes` should be at the same level as `coverage:status:project`, i.e. `coverage:status:changes`.